plugin.video.locast/resources/lib/net.py

```diff
@@ -0,0 +1,347 @@
+#   Copyright (C) 2018 Lunatixz
```

Comment:
maybe at some point creating a library with this would save time reviewing the same file over and over again :)